client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -34,17 +34,17 @@
 
 // Processing Copy Site Styles
 .copy-site.processing-copy {
-	.processing-step, .stepper-loader {
+	.processing-step, .loading {
```

Comment:
I think this one still needs to be renamed?

Reply:
🤦 Done!